src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1370,6 +1381,45 @@ def add_units_table_to_nwbfile(
     waveform_sds: Optional[np.ndarray] = None,
     unit_electrode_indices: Optional[list[list[int]]] = None,
     null_values_for_properties: Optional[dict] = None,
+):
+    """
+    add unist table will become a private method, use `add_sorting_to_nwbfile` instead.
+    """
+
+    warnings.warn(
+        "This function is deprecated and will be removed in or after October 2025. "
+        "Use the 'add_sorting_to_nwbfile' function instead.",
+        DeprecationWarning,
+    )
+    _add_units_table_to_nwbfile(
+        sorting=sorting,
+        nwbfile=nwbfile,
+        unit_ids=unit_ids,
+        property_descriptions=property_descriptions,
+        skip_properties=skip_properties,
+        units_table_name=units_table_name,
+        unit_table_description=unit_table_description,
+        write_in_processing_module=write_in_processing_module,
+        waveform_means=waveform_means,
+        waveform_sds=waveform_sds,
+        unit_electrode_indices=unit_electrode_indices,
+        null_values_for_properties=null_values_for_properties,
+    )
+
+
+def _add_units_table_to_nwbfile(
+    sorting: BaseSorting,
+    nwbfile: pynwb.NWBFile,
+    unit_ids: Optional[list[Union[str, int]]] = None,
+    property_descriptions: Optional[dict] = None,
+    skip_properties: Optional[list[str]] = None,
+    units_table_name: str = "units",
+    unit_table_description: Optional[str] = None,
+    write_in_processing_module: bool = False,
+    waveform_means: Optional[np.ndarray] = None,
+    waveform_sds: Optional[np.ndarray] = None,
+    unit_electrode_indices: Optional[list[list[int]]] = None,
+    null_values_for_properties: Optional[dict] = None,
```

Comment:
Haven't checked everywhere, but looks like the docstring is missing for `null_values_for_properties` in `add_sorting_to_nwbfile` as well

Reply:
Added them.